piecrust/src/store.rs

```diff
@@ -217,20 +237,13 @@ fn commit_from_dir<P: AsRef<Path>>(
 
         // SAFETY it is safe to deserialize the file here, since we don't use
         // the module here. We just want to check if the file is valid.
-        if unsafe {
-            Module::deserialize_file(engine, &objectcode_path).is_err()
-        } {
+        if Module::from_file(engine, &objectcode_path).is_err() {
             let bytecode = Bytecode::from_file(bytecode_path)?;
             let module =
                 Module::new(engine, bytecode.as_ref()).map_err(|err| {
                     io::Error::new(io::ErrorKind::InvalidData, err)
                 })?;
-            fs::write(
-                objectcode_path,
-                module.serialize().map_err(|err| {
-                    io::Error::new(io::ErrorKind::InvalidData, err)
-                })?,
-            )?;
+            fs::write(objectcode_path, module.serialize())?;
```

Comment:
since you are replacing `objectcode` with `module` maybe also rename the `objectcode_path` variable? Does something like `modulecode_path` make sense?

Reply:
You're right. I switched the nomenclature to `module` - without "code".